### ISSUE TITLE
LOCATE depends on the current directory

### ISSUE THREAD

**Issue by ruv** [forthy42/gforth#19]
When Gforth is started from the binary that is located in the git repository working directory (project root), but the current directory is not the project root — `locate` and `edit` don't work properly.
Namely, `locate` doesn't show context in the source file, and `edit` doesn't open the proper source file.

Changing the current directory via `set-dir` also affects these words.

**Comment by forthy42** [comment]
Yes, this is likely a wontfix. You are supposed to use make install to generate binaries with absolute paths. 

Am 12. November 2019 23:12:58 MEZ schrieb ruv <notifications@github.com>:
>When Gforth is started from the binary that is located in the git
>repository working directory (project root), but the current directory
>is not the project root — `locate` and `edit` don't work properly.>
>Namely, `locate` doesn't show context in the source file, and `edit`
>doesn't open the proper source file.>
>>
>Changing the current directory via `set-dir` also affects these words.>
>>
>>
>-- >
>You are receiving this because you are subscribed to this thread.>
>Reply to this email directly or view it on GitHub:>
>https://github.com/forthy42/gforth/issues/19

-- 
Bernd Paysan
"If you want it done right, you have to do it yourself"
net2o ID: kQusJzA;7*?t=uy@X}1GWr!+0qqp_Cn176t4(dQ*
http://bernd-paysan.de/
